src/tests/ftest/soak/soak.py

```diff
@@ -120,65 +119,192 @@ def remote_copy(self, hostlist, remote_dir, local_dir):
                     ", ".join(
                         [str(result[key]) for key in result if key != 0])))
 
-    def create_ior_cmdline(self, job_params, job_spec, pool):
+    def is_harasser(self, harasser):
+        """Check if harasser is defined in yaml.
+
+        Args:
+            harasser (list): list of harassers to launch
+
+        Returns: bool
+
+        """
+        return self.h_list and harasser in self.h_list
+
+    def launch_harassers(self, harassers, pools):
+        """Launch any harasser tests if defined in yaml.
+
+        Args:
+            harasser (list): list of harassers to launch
+            pools (TestPool): pool obj
+
+        """
+        job = []
+        # Launch harasser after one complete pass
+        for harasser in harassers:
+            if "rebuild" == harasser:
```

Comment:
(pylint-misplaced-comparison-constant) Comparison should be harasser == 'rebuild'